.github/workflows/ci.yml

```diff
@@ -125,10 +125,10 @@ jobs:
         run: |
           tox -e fast
           npm run test --coverage
-        if: matrix.python-version != 3.10
+        if: matrix.python-version != 3.10 && matrix.python-version != 3.11
```

Comment:
See above - I think we'll want to run fast tests for 3.10

Reply:
Running fast tests for 3.10 now